monitoring/storage_linux.go

```diff
@@ -155,30 +159,48 @@ func (c *storageChecker) checkHighWatermark(ctx context.Context, reporter health
 		return trace.BadParameter("disk capacity at %v is 0", c.path)
 	}
 	checkerData := HighWatermarkCheckerData{
-		HighWatermark:  c.HighWatermark,
-		Path:           c.Path,
-		TotalBytes:     totalBytes,
-		AvailableBytes: availableBytes,
+		WatermarkCritical: c.HighWatermark,
+		WatermarkWarning:  c.HighWatermark - 10, // Set warning watermark 10% below the critical watermark
```

Comment:
I'd just have 2 separate constants TBH for soft/hard limit. Is there any particular reason you wanted to "tie" them one to another this way?

Reply:
We want the watermark to be configurable/overrideable in emergency situations. If the values are separately configurable, we might run into incorrect behavior if the critical watermark is set to a lower value than the warning watermark. Having them tied together will make sure we don't run into that situation.

Reply:
This scenario can be just checked during checker initialization, right? In Check(), as we usually do. And just return an error if that's the case.

Reply:
Yeah, that works. I guess it'd be nicer to have separate control over the two threshold values.